hmda-regulator/src/main/resources/application.conf

```diff
@@ -44,38 +46,36 @@ hmda {
 }
 
 
-aws {
-  access-key-id = ""
-  access-key-id = ${?AWS_ACCESS_KEY_ID}
-  secret-access-key = ""
-  secret-access-key = ${?AWS_SECRET_ACCESS_KEY}
-  environment = "dev" //change to "prod" for production S3 bucket
-  environment = ${?AWS_ENVIRONMENT}
+private-aws {
+  private-access-key-id = ""
+  private-access-key-id = ${?AWS_ACCESS_KEY_ID}
+  private-secret-access-key = ""
+  private-secret-access-key = ${?AWS_SECRET_ACCESS_KEY}
+  private-environment = "dev"
+  private-environment = ${?AWS_ENVIRONMENT}
   private-s3-bucket = "cfpb-hmda-export"
   private-s3-bucket = ${?S3_PRIVATE_BUCKET}
-  region = "us-east-1"
-  region = ${?AWS_REGION}
-  year ="2018"
+  private-region = "us-east-1"
+  private-region = ${?AWS_REGION}
+  private-year ="2018"
 }
 
 public-aws {
   public-access-key-id = ""
   public-access-key-id = ${?AWS_ACCESS_KEY_ID}
   public-secret-access-key = ""
   public-secret-access-key = ${?AWS_SECRET_ACCESS_KEY}
-  public-environment = "dev" //change to "prod" for production S3 bucket
+  public-environment = "dev"
   public-environment = ${?AWS_ENVIRONMENT}
   public-s3-bucket = "cfpb-hmda-export"
   public-s3-bucket = ${?S3_PRIVATE_BUCKET}
```

Comment:
`public-s3-bucket` should be `cfpb-hmda-public` and env variable should be `S3_PUBLIC_BUCKET`